build/package/Dockerfile

```diff
@@ -8,14 +8,14 @@ RUN make clean && make build
 
 # ---
 
-FROM alpine:3.13
+FROM alpine:latest
```

Comment:
latest sounds cool, but can it guarantee stability?

Reply:
The release strategy of Alpine Linux seems promising: https://alpinelinux.org/releases/